### PR TITLE
Cornerdetection

### PR DESCRIPTION
**Pull request by aryannarora**
Corner Detection implemented.

### CODE REVIEW

File: corner detection/corner detection.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import numpy as np
+## pixel values takes bgr as params, not rgb.
+img = cv2.imread('corners.jpg')
+## cvtColor = convert color
+grayed = cv2.cvtColor(img,cv2.COLOR_BGR2GRAY)
+
+##float32 is used for precision.
+grayed = np.float32(grayed)
+
+## goodFeaturesToTrack(imgSource, maxNumberOfCorners, Quality, MinimumDistBetweenCorners)
```
Comment (utkarshmttl) [inline corner detection/corner detection.py]: Add the meaning of input parameters.

**Comment by utkarshmttl** [comment]
Good work!
Also, try to add a result on the input image as given in Readme.
Also, convert as a function as mentioned in #1

**Comment by aryannarora** [comment]
added new file for function.
added ouput for sample input in readme.

**Comment by utkarshmttl** [comment]
May you please remove the editor-dependent (non essential) files?

**Comment by aryannarora** [comment]
Done

**Comment by utkarshmttl** [comment]
Is the only difference between corner detection and corner detection function that one is in function form and other isn't? Delete the non-function one if yes.

**Pull request merged**